app/controllers/foreman_inventory_upload/uploads_controller.rb

```diff
@@ -24,6 +24,21 @@ def throw_flash_error(message)
       )
     end
 
+    def enable_cloud_connector
+      cloud_connector = ForemanRhCloud::CloudConnector.new
+      cloud_connector.install
+    end
+
+    def cloud_connector_status
+      cloud_connector = ForemanRhCloud::CloudConnector.new
+
+      if (job = cloud_connector.running_job)
+        job.task
+      else
+        cloud_connector.latest_job
```

Comment:
Also can you change this to :
```
else
        cloud_connector.latest_job
elif
 render json something
 ```
 When cloud_connector.latest_job is null the server returns an error for me